src/control/common/collection_utils.go

```diff
@@ -170,3 +170,20 @@ func PercentageString(part, total uint64) string {
 	return fmt.Sprintf("%v %%",
 		int((float64(part)/float64(total))*float64(100)))
 }
+
+// TokenizeCommandSeparatedString splits the input string on comma boundaries,
+// and discards any empty strings in the result.
+func TokenizeCommaSeparatedString(str string) []string {
```

Comment:
TokenizeCommandSeparatedString should be TokenizeCommaSeparatedString

Reply:
Good catch, fingers move faster than the mind sometimes. :)